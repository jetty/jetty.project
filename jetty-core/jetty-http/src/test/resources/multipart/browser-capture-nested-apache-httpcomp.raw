--Cku4UvJrPFCXkXjge2a2Y2sgq1bbOa
Content-Disposition: form-data; name="reporter"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

<user@company.com>
--Cku4UvJrPFCXkXjge2a2Y2sgq1bbOa
Content-Disposition: form-data; name="timestamp"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

2018-03-21T18:52:18+00:00
--Cku4UvJrPFCXkXjge2a2Y2sgq1bbOa
Content-Disposition: form-data; name="comments"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

this couldn't be parsed
--Cku4UvJrPFCXkXjge2a2Y2sgq1bbOa
Content-Disposition: form-data; name="attachment"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

--nM8_n8ugj9L3fIomqyU6h9Wpb6Wt-3w
Content-Disposition: form-data; name="fruit"

banana
--nM8_n8ugj9L3fIomqyU6h9Wpb6Wt-3w
Content-Disposition: form-data; name="color"

yellow
--nM8_n8ugj9L3fIomqyU6h9Wpb6Wt-3w
Content-Disposition: form-data; name="cost"

$0.12 USG
--nM8_n8ugj9L3fIomqyU6h9Wpb6Wt-3w
Content-Disposition: form-data; name="comments"

--divc688gD49-GaZcLkprfUb8-PWOjF3Z
--nM8_n8ugj9L3fIomqyU6h9Wpb6Wt-3w--

--Cku4UvJrPFCXkXjge2a2Y2sgq1bbOa--
