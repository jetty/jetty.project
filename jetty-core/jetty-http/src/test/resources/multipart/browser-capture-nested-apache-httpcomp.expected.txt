Request-Header|Accept-Encoding|gzip,deflate
Request-Header|Connection|keep-alive
Request-Header|Content-Length|1203
Request-Header|Content-Type|multipart/form-data; boundary=Cku4UvJrPFCXkXjge2a2Y2sgq1bbOa
Request-Header|Host|localhost:9090
Request-Header|User-Agent|Apache-HttpClient/4.5.5 (Java/1.8.0_162)
Request-Header|X-BrowserId|apache-httpcomp
Parts-Count|4
Part-ContainsContents|reporter|<user@company.com>
Part-ContainsContents|timestamp|2018-03-21T18:52:18+00:00
Part-ContainsContents|comments|this couldn't be parsed
Part-ContainsContents|attachment|banana